code/game/objects/effects/portals.dm

```diff
@@ -50,6 +50,17 @@
 	creator = null
 	return ..()
 
+/obj/effect/portal/proc/blockExiles(atom/movable/AM)

```

Comment:
The duplication of this check is also bad, don't duplicate this create a generic proc in helpers.dm